### PR TITLE
Fabo/eslint pretty

### PR DESCRIPTION
**Pull request by faboweb**
Fixed the compatibility between eslint and prettier

Missing:
- [x] fix linting issues from updated packages
- [x] add Karolys rules
- [x] add `eslint --fix` to `yarn lint:fix`

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -2,7 +2,12 @@
 
 module.exports = {
   root: true,
-  extends: ["plugin:vue/recommended", "prettier", "prettier/vue"],
+  extends: [
+    "plugin:vue/recommended",
```
Comment (sabau) [inline .eslintrc.js]: in this way we override again the rules, basically this won't change prettier behaviour much

Comment (faboweb) [reply .eslintrc.js]: I thought the wished outcome is to fix the html in the vue templates. This is now the case.

File: app/src/renderer/components/common/AnchorCopy.vue [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <template>
-  <a class="anchor-copy" @click="click"
```
Comment (faboweb) [inline app/src/renderer/components/common/AnchorCopy.vue]: @sabau see here

File: app/src/renderer/components/common/TmBtn.vue [hunk 3]
```diff
@@ -1,36 +1,32 @@
...
+        {{ icon }}
+      </i>
+      <img
         v-else-if="img"
         :src="img"
         :class="'tm-btn__img'"
         aria-hidden="true"
-      /><span v-if="value" class="tm-btn__value">{{ value }}</span></span
-    >
+      /><span v-if="value" class="tm-btn__value">{{ value }}</span>
```
Comment (sabau) [inline app/src/renderer/components/common/TmBtn.vue]: It improves but is still closing/opening sibilings

Comment (faboweb) [reply app/src/renderer/components/common/TmBtn.vue]: What would you expect here?

**Comment by fedekunze** [comment]
unit tests failing

**Pull request closed without merging**